docs/devel/contributing.md

```diff
@@ -115,8 +115,7 @@ files at this time, but it's very easy to create run/debug targets and use the d
 
 1. Clone nuclio `git clone git@github.com:nuclio/nuclio.git > ~/nuclio`
 2. Open GoLand and **File > Open** and select `~/nuclio`
-3. Enable go modules **GoLand > Preferences > Go > Go Modules (vgo)** and ensure `Enable Go Modules` box is checked
```

Comment:
Miscommunication on my part...
The whole line should not be removed, only the `(vgo)` word

Reply:
@TomerShor returned it